### ISSUE TITLE
Support CD Audio (CDDA) for disc images that contain it

### ISSUE THREAD

**Issue by NintendoManiac64** [sysprogs/WinCDEmu#29]
Many older games from before year 2000 used CD audio (CDDA) for their music, the same thing you'd find with an actual music CD (it was to the point that you could put something like the Quake II game CD into a CD player and it'd actually play the music).

WinCDEmu does not seem to support this form of CD Audio which is unfortunate given that many of these older games are exactly the type of software that would require mounting of CD images in the first place.

According to the below-linked forum thread, Daemon Tools apparently supports handling CD Audio on mounted disc images (and from my testing, Daemon Tools Lite as well):
https://sysprogs.com/w/forums/topic/cd-images-with-audio-tracks/

Also in that thread, at least the following games are known to use CD Audio for their music:

Pro Pinball : Timeshock !
Earthworm Jim
Pitfall: The Mayan Adventure

And from my own experience I can add Quake II, AstroRock, and Tsukihime to that list.

Speaking of Tsukihime, many circa-2000 and older Japanese PC visual novels similarly use CD Audio such as the aforementioned Tsukihime (by Type-Moon who are more well known for their later "Fate" works, e.g. Fate/stay night, Fate/Zero, and Fate/Grand Order).

**Comment by seangelos** [comment]
Yup, Quake does this as well, and it's impossible to get the soundtrack to play otherwise. It would be really nice to see this added, as a lot of older games depend on it.

**Comment by langyxxl** [comment]
linux cdemu can support audio tracks, maybe some code can copy from it.

**Comment by expected-ingot** [comment]
bump

**Comment by UnluckyForSome** [comment]
Bump +1year
